scripts/hudson/narayana.sh

```diff
@@ -985,10 +993,10 @@ export ANT_OPTS="$ANT_OPTS $IPV6_OPTS"
 
 # run the job
 
-[ $NARAYANA_BUILD = 1 ] && build_narayana "$@"
-[ $AS_CLONE = 1 ] && clone_as "$@"
+[ $NARAYANA_BUILD = 1 ] && build_narayana "$@" # NB we should do this after building the AS since it takes longer
+[ $AS_CLONE = 1 ] && AS_BUILD=1 # keep it simple and assume that we are cloning the AS because we want to build it
+[ $AS_DOWNLOAD = 1 ] && AS_BUILD=1 # keep it simple and build it from scratch - we can revisit it later if required
```

Comment:
I think this is confusing, I think you could just "re-program" the PROFILE for those that have AS_DOWLOAD=1 to be AS_CLONE=1 and AS_BUILD=1 instead

Reply:
It seems it's only the LRA PROFILE: https://github.com/jbosstm/narayana/blob/master/scripts/hudson/narayana.sh#L192

Reply:
If you change
`export AS_BUILD=0 AS_CLONE=0 AS_DOWNLOAD=1 AS_TESTS=0 NARAYANA_BUILD=0 NARAYANA_TESTS=0 XTS_AS_TESTS=0 XTS_TESTS=0 TXF_TESTS=0 txbridge=0`
to:
`export AS_BUILD=1 AS_CLONE=1 AS_DOWNLOAD=0 AS_TESTS=0 NARAYANA_BUILD=0 NARAYANA_TESTS=0 XTS_AS_TESTS=0 XTS_TESTS=0 TXF_TESTS=0 txbridge=0` I think it would accomplish what you are trying to do with this change

Reply:
Yes, I noticed that, the last push did something similar to what you suggested but I will take another look ...